.circleci/config.yml

```diff
@@ -0,0 +1,49 @@
+version: 2
+jobs:
+  build:
+
+    docker:
+      - image: circleci/node:8
+
+    environment:
+      TZ: /usr/share/zoneinfo/America/Los_Angeles
+      YARN_VERSION: 1.2.1
+      TRAVIS_REPO_SLUG: facebook/react
+
+    parallelism: 4
+
+    steps:
+      - checkout
+
+      - run: echo $CIRCLE_COMPARE_URL | cut -d/ -f7
+
+      - run:
+          name: Install Yarn
+          command: |
+            if [[ ! -e ~/.yarn/bin/yarn || $(yarn --version) != "${YARN_VERSION}" ]]; then
+              curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
+            fi
+
+      - restore_cache:
+          key: node-modules-{{ checksum "yarn.lock" }}
+
+      - run:
+          name: Nodejs Version
+          command: node --version
+
+      - run:
+          name: Install Packages
+          command: yarn install
+
+      - run:
+          name: Test Packages
+          command: ./scripts/circleci/test_entry_point.sh
+
+      - save_cache:
+          key: node-modules-{{ checksum "yarn.lock" }}
```

Comment:
probably worth prefixing/appending `-v1` to the key or something so it can be neatly manually invalidated if required